src/main/java/fi/aalto/cs/apluscourses/ui/ideactivities/PopupTransparencyHandler.java

```diff
@@ -0,0 +1,65 @@
+package fi.aalto.cs.apluscourses.ui.ideactivities;
+
+public class PopupTransparencyHandler {
```

Comment:
"Popup" in the name is perhaps unnecessary because this isn't popup specific (except for the parameter of `update(boolean)` but that could perhaps, in this class' context, be called something like `increaseOpaquity`).

Reply:
You are correct